.github/workflows/ai-codereview.yml

```diff
@@ -0,0 +1,61 @@
+name: AI code review
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: Mirantis/ai-ci-scripts
```

Comment:
this does not seem to be a [working step](https://github.com/k0rdent/kcm/actions/runs/13845349129/job/38770623599?pr=1193)